BlogTemplate/Pages/Drafts.cshtml

```diff
@@ -4,9 +4,13 @@
 @{
     ViewData["Title"] = "Drafts";
 }
-<h2>@ViewData["Title"]</h2>
+<h2>@ViewData["Title"]</h2>
+
+@if (!Model.Posts.Any())

```

Comment:
this if could go into the code block with the foreach, and the foreach could be put inside an else block (same comments as below)